### ISSUE TITLE
efsVolumeConfiguration for ECS task definition

### ISSUE THREAD

**Issue by avoidik** [terraform-providers/terraform-provider-aws#11696]
Hello,

Please consider adding `efsVolumeConfiguration` to `volume` block of `aws_ecs_task_definition` resource definition. 

```json
    "volumes": [
        {
            "name": "efs-html",
            "efsVolumeConfiguration": {
                "fileSystemId": "fs-1234",
                "rootDirectory": "/path/to/my/data"
            }
        }
    ],
```

https://aws.amazon.com/about-aws/whats-new/2020/01/amazon-ecs-preview-support-for-efs-file-systems-now-available/

**Comment by avoidik** [comment]
https://github.com/aws/containers-roadmap/issues/53#issuecomment-575807680

**Comment by bflad** [comment]
This functionality is included with AWS Go SDK version 1.28.5, already merged (https://github.com/terraform-providers/terraform-provider-aws/pull/11630), and will release with version 2.46.0 of the Terraform AWS Provider on Thursday. 👍

**Comment by avoidik** [comment]
@bflad shall I create PR to improve documentation? :)

**Comment by bflad** [comment]
Hi folks 👋 My apologies on this one -- it will require some implementation and documentation work in the Terraform resource. Reopening! Here's the related pull request: https://github.com/terraform-providers/terraform-provider-aws/pull/11707

**Comment by bflad** [comment]
Support for this new EFS volume configuration has been merged and will release with version 2.46.0 of the Terraform AWS Provider, tomorrow. Thanks to @DrFaust92 for the implementation. 👍
